test/unit/stream_alert_rule_processor/test_classifier.py

```diff
@@ -13,148 +13,263 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 '''
-
-# command: nosetests -v -s test/unit/
-# specific test: nosetests -v -s test/unit/file.py:TestStreamPayload.test_name
-
-import base64
 import json
 
-from nose.tools import assert_equal, assert_not_equal
+from mock import call, patch
 
-import stream_alert.rule_processor.classifier as sa_classifier
+from nose.tools import (
+    assert_equal,
+    assert_false,
+    assert_is_instance,
+    assert_list_equal,
+    assert_true
+)
 
-from stream_alert.rule_processor.classifier import StreamPayload, StreamClassifier
-from stream_alert.rule_processor.pre_parsers import StreamPreParsers
+import stream_alert.rule_processor.classifier as sa_classifier
 from stream_alert.rule_processor.config import load_config
+from stream_alert.rule_processor.payload import load_stream_payload
+
+from unit.stream_alert_rule_processor.test_helpers import (
+    _make_kinesis_raw_record
+)
 
 
-class TestStreamPayload(object):
-    """Test class for StreamPayload"""
+class TestStreamClassifier(object):
+    """Test class for StreamClassifier"""
+
     def __init__(self):
-        self.config = {}
+        self.classifier = None
```

Comment:
Does `pylint` complain if you leave this method out? No need to initialize it here since it's set in `setup`.